File: AllMyBooksArePacked/Book.java
public class Book{
  public String title;
  public String author;
  public String price;
  public String weight;
  public String isbn; 
  
  /**
   * Constructor
   * Every book element needs to contain these five fields
   */
  public Book(String title, String author, String price, String weight, String isbn){
    this.title = title;
    this.author = author;
    this.price = price;
    this.weight = weight;
    this.isbn = isbn;
  }
}

File: AllMyBooksArePacked/src/Book.js
'use strict';

class Book {
	constructor(title, author, price, weight, isbn10) {
		this.title = title;
		this.author = author;
		this.price = price;
		this.weight = weight;
		this.isbn10 = isbn10;
	}
	getObject() {
		return {
			title: this.title,
			price: this.price,
			author: this.author,
			weight: this.weight,
			isbn10: this.isbn10
		}
	}
}

module.exports = {
	Book
}
